force-app/main/default/lwc/serviceScheduleReview/serviceScheduleReview.html

```diff
@@ -0,0 +1,131 @@
+                &nbsp;&ndash;&nbsp;
+                <lightning-formatted-date-time
+                    value={lastSessionEndDateTime}
+                ></lightning-formatted-date-time>
+            </c-output-field>
+
+            <!-- times -->
+            <c-output-field label="Time">
+                <lightning-icon
+                    icon-name="utility:clock"
+                    alternative-text=""
+                    size="x-small"
+                    class="slds-p-right_x-small"
+                >
+                </lightning-icon>
+                <lightning-formatted-date-time
+                    value={firstSessionStartDateTime}
+                    hour="2-digit"
+                    minute="2-digit"
+                    time-zone-name="short"
+                    hour12="true"
+                ></lightning-formatted-date-time>
+                <!-- spacing and en-dash between times -->
+                &nbsp;&ndash;&nbsp;
+                <lightning-formatted-date-time
+                    value={lastSessionEndDateTime}
+                    hour="2-digit"
+                    minute="2-digit"
+                    time-zone-name="short"
+                    hour12="true"
+                ></lightning-formatted-date-time>
+            </c-output-field>
+        </lightning-layout-item>
+
+        <!-- Column 2 -->
+        <lightning-layout-item size="6">
+            <c-output-field
+                label={schedule.fields.PrimaryServiceProvider__c.label}
+                if:true={schedule.record.PrimaryServiceProvider__c}
+            >
+                <lightning-icon
+                    icon-name="utility:user"
+                    alternative-text=""
+                    size="x-small"
+                    class="slds-p-right_x-small"
+                >
+                </lightning-icon>
+                {schedule.record.PrimaryServiceProvider__c.Name}
+            </c-output-field>
+            <c-output-field
+                label={schedule.fields.OtherServiceProvider__c.label}
+                if:true={schedule.record.OtherServiceProvider__c}
+            >
+                <lightning-icon
+                    icon-name="utility:user"
+                    alternative-text=""
+                    size="x-small"
+                    class="slds-p-right_x-small"
+                >
+                </lightning-icon>
+                {schedule.record.OtherServiceProvider__c.Name}
+            </c-output-field>
+            <c-output-field
+                label={schedule.fields.ParticipantCapacity__c.label}
+                if:true={schedule.record.ParticipantCapacity__c}
+            >
+                {schedule.record.ParticipantCapacity__c}
+            </c-output-field>
+        </lightning-layout-item>
+    </lightning-layout>
+
+    <lightning-accordion
+        active-section-name={activeSections}
+        allow-multiple-sections-open
+    >
+        <!--{labels.participants}-->
```

Comment:
It looks like the mockup has a lightning card around each datatable and the number of records after the title for each table. Similar to a related list.

Reply:
re: card -- I'm using the base components for accordion and accordion-section here, so this is how they render. we can review with ashwin and revisit if needed. 
re: number -- missed this and could add in iteration. shouldn't be an issue and would make sense to add when we are querying/using real data.